analysis/import/TokeiImporter/src/test/resources/tokei_without_inner.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Markdown": {
+    "blanks": 0,
+    "code": 1965,
+    "comments": 0,
+    "reports": [
+      {
+        "stats": {
+          "blanks": 0,
+          "code": 450,
+          "comments": 0,
+          "blobs": {}
```

Comment:
Example for blobs `"blobs": { "Bash": { "blanks": 0, "code": 2, "comments": 3, "blobs": {} } }`